feature/experimental/tunnel/otg_tests/tunnel_interface_based_resize_test/README.md

```diff
@@ -0,0 +1,90 @@
+# TUN-1.6: Tunnel End Point Resize - Interface Based GRE Tunnel
+
+## Summary
+* Tunnel End Point Resize test for Interface Based GRE Tunnels to ensure inflight increase or decrease in tunnel endpoints do not cause packet drops. The test will be conducted for both Encapsulation as well as decapsulation of packets.
+
+## Topology
+
+```mermaid
+graph LR; 
+        A[OTG:Port1] --> B[Port1:DUT:Port2\lag1];
+        B --> C[Port2\lag1:OTG];
+        D[DUT:Port3\lag1] --> E[Port3\lag1:OTG];
+        f[DUT:Port4\lag2] --> G[Port4\lag2:OTG];
+        H[DUT:Port5\lag2] --> I[Port5\lag2:OTG];
+        J[DUT:Port6] --> K[Port6:OTG];
+```
+
+
+
+## SUB-Test:1 Encap Test
+*   The test should configure 32 GRE tunnel interfaces each with different /32 and /128 tunnel destination prefixes of the same /27 and /64 prefix range. Each of these tunnel interfaces should use the loopback address as the source-address for their respective address-families.
+ *   Configure an IPv4 and an IPv6 address for each of the 32xtunnel interfaces.
+ *   Add 32 different v4 and v6 static routes for the same /24 IPv4 and /64 IPv6 destination prefixes. These static routes should point at the 32 different tunnel interface addresses by their respective address families as next-hop.
```

Comment:
If we are talking about Cisco's implementation they are the same static route pointing to NH tunnel interface. eg
  10.255.255.255/32 tunnel-ip1200
  10.255.255.255/32 tunnel-ip1201
  10.255.255.255/32 tunnel-ip1202
  10.255.255.255/32 tunnel-ip1203
  ...
  
  Also why are you mentioning /24 here?

Reply:
There are 2 types of static routes here per address family

1. /24 and /64 static routes for the inner payload pointing at the tunnel interfaces
2. /27 and /64 static routes for the tunnel destination addresses.

The mention here is for "1".

Btw, I made some changes to the test. PTAL